src/app/backend/api/types.go

```diff
@@ -157,28 +159,29 @@ var KindToAPIMapping = map[string]struct {
-	ResourceKindPersistentVolume:        {"persistentvolumes", ClientTypeDefault, false},
-	ResourceKindPod:                     {"pods", ClientTypeDefault, true},
-	ResourceKindReplicaSet:              {"replicasets", ClientTypeExtensionClient, true},
-	ResourceKindReplicationController:   {"replicationcontrollers", ClientTypeDefault, true},
-	ResourceKindResourceQuota:           {"resourcequotas", ClientTypeDefault, true},
-	ResourceKindSecret:                  {"secrets", ClientTypeDefault, true},
-	ResourceKindService:                 {"services", ClientTypeDefault, true},
-	ResourceKindStatefulSet:             {"statefulsets", ClientTypeAppsClient, true},
-	ResourceKindThirdPartyResource:      {"thirdpartyresources", ClientTypeExtensionClient, true},
-	ResourceKindStorageClass:            {"storageclasses", ClientTypeStorageClient, false},
-	ResourceKindEndpoint:                {"endpoints", ClientTypeDefault, true},
+	ResourceKindConfigMap:                {"configmaps", ClientTypeDefault, true},
+	ResourceKindDaemonSet:                {"daemonsets", ClientTypeExtensionClient, true},
+	ResourceKindDeployment:               {"deployments", ClientTypeExtensionClient, true},
+	ResourceKindEvent:                    {"events", ClientTypeDefault, true},
+	ResourceKindHorizontalPodAutoscaler:  {"horizontalpodautoscalers", ClientTypeAutoscalingClient, true},
+	ResourceKindIngress:                  {"ingresses", ClientTypeExtensionClient, true},
+	ResourceKindJob:                      {"jobs", ClientTypeBatchClient, true},
+	ResourceKindLimitRange:               {"limitrange", ClientTypeDefault, true},
+	ResourceKindNamespace:                {"namespaces", ClientTypeDefault, false},
+	ResourceKindNode:                     {"nodes", ClientTypeDefault, false},
+	ResourceKindPersistentVolumeClaim:    {"persistentvolumeclaims", ClientTypeDefault, true},
+	ResourceKindPersistentVolume:         {"persistentvolumes", ClientTypeDefault, false},
+	ResourceKindPod:                      {"pods", ClientTypeDefault, true},
+	ResourceKindReplicaSet:               {"replicasets", ClientTypeExtensionClient, true},
+	ResourceKindReplicationController:    {"replicationcontrollers", ClientTypeDefault, true},
+	ResourceKindResourceQuota:            {"resourcequotas", ClientTypeDefault, true},
+	ResourceKindSecret:                   {"secrets", ClientTypeDefault, true},
+	ResourceKindService:                  {"services", ClientTypeDefault, true},
+	ResourceKindStatefulSet:              {"statefulsets", ClientTypeAppsClient, true},
+	ResourceKindThirdPartyResource:       {"thirdpartyresources", ClientTypeExtensionClient, true},
+	ResourceKindCustomResourceDefinition: {"CustomResourceDefinition", ClientTypeAPIExtensionsClient, true},
+	ResourceKindStorageClass:             {"storageclasses", ClientTypeStorageClient, false},
+	ResourceKindEndpoint:                 {"endpoints", ClientTypeDefault, true},
```

Comment:
Be careful during merge if cron jobs will go to master first. It will not merge automatically due to indent change.